packages/e2e-tests/puppeteer/helpers.ts

```diff
@@ -250,7 +250,11 @@ export async function waitAndApproveDepositWithHub(
 ): Promise<void> {
   console.log('Making deposit with hub');
   const walletIFrame = page.frames()[1];
-  await walletIFrame.waitForSelector('#please-approve-transaction', {timeout: TX_WAIT_TIMEOUT}); // longer timeout here because blockchain is slow
+  console.log('Waiting for #please-approve-transaction');
+  console.time('#please-approve-transaction');
+  await walletIFrame.waitForSelector('#please-approve-transaction', {timeout: TX_WAIT_TIMEOUT * 3}); // longer timeout here because blockchain is slow
```

Comment:
Why would this transaction take 3x longer than any others? Would a change to `TX_WAIT_TIMEOUT` be more appropriate?